overlays/update-images/set-repo-registry-tag.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+set -euf -o pipefail
+echo "running in $(dirname "${BASH_SOURCE[0]}")"
+cd "$(dirname "${BASH_SOURCE[0]}")"
+
+for arg in "$@"; do
+  if [ "$arg" == "--help" ] || [ "$arg" == "-h" ]; then
+    echo 'Usage: ./set-repo-registry.sh REGISTRY REPO TAG'
+    exit 1
+  fi
+done
+
+registry="${1:-index.docker.io}"
+repository="${2:-sourcegraph}"
```

Comment:
Can we just use https://github.com/sourcegraph/deploy-sourcegraph/tree/master/tools ?